code/project.clj

```diff
@@ -10,7 +10,13 @@
             :url "http://www.apache.org/licenses/LICENSE-2.0.txt"
             :distribution :repo}
 
-  :plugins [[lein-parent "0.3.5"]]
+  :plugins [[lein-parent "0.3.5"]
```

Comment:
I think it's better to have the plugins in parent-project. Such way it will be updated when we update parent project

Reply:
Yes, agreed! This is one of the topics I wanted to discuss.

Reply:
The plugins were moved to parent.